api/filter_outputs.go

```diff
@@ -418,3 +419,50 @@ func buildDownloadsObject(previousFilterOutput, filterOutput *models.Filter, dow
 		}
 	}
 }
+
+func (api *FilterAPI) addEventHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	filterOutputID := vars["filter_output_id"]
+
+	logData := log.Data{"filter_output_id": filterOutputID}
+	log.Info("add event to filter output endpoint called", logData)
+
+	defer r.Body.Close()
+
+	bytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
```

Comment:
Do we not audit action?

Reply:
I considered adding auditing but realised this endpoint will never be called with a user. It is only called from the exporter services which are kafka driven and have no auth. Do you think its worth adding for completeness?

Reply:
So I thought as much and was thinking maybe its worth adding just so its all tied up neatly and probably more importantly is that currently this is still accessible to a florence user until we add permissions to users.

Lastly we do not know what services (maybe api's) might call this endpoint in the future.

Maybe open a discussion with ELeanor and Ian (when he gets back from holiday).

I will approve for now though.